src/components/pages/exportcontent/contentcatalogue.js

```diff
@@ -0,0 +1,30 @@
+import React, { useState, useEffect } from 'react';
+import { useLocation } from '@reach/router';
+import { buildQueryString, parseQueryString } from '~utils/query-string';
+
+const SIMPLESEARCHURL = `https://us-east-1.aws.webhooks.mongodb-realm.com/api/client/v2.0/app/devhub-search-service-dxfcg/service/devhub_export/incoming_webhook/simplesearch`;
+const COMPOSITESEARCHURL = `https://us-east-1.aws.webhooks.mongodb-realm.com/api/client/v2.0/app/devhub-search-service-dxfcg/service/devhub_export/incoming_webhook/compositesearch`;
+
+const ContentCatalogue = () => {
+    const { search } = useLocation();
+    const { ...params } = parseQueryString(search);
+    const queryParams = buildQueryString(params);
+    const urlEndPoint =
+        'content' in params ? SIMPLESEARCHURL : COMPOSITESEARCHURL;
+    const [searchResult, setSearchResult] = useState(null);
+    useEffect(() => {
+        const url = `${urlEndPoint}${queryParams}`;
```

Comment:
Does this need to be in the `useEffect`? (Since we only run this `useEffect` on the component mount?)